template/src/App.tsx

```diff
@@ -1,19 +1,22 @@
 import React from 'react'
+import { useTranslation } from 'react-i18next'
 
 import logo from './assets/images/logo.svg'
 import './dummy.scss'
 import './assets/stylesheets/application.scss'
 
 function App(): JSX.Element {
+  const { t } = useTranslation()
+
   return (
     <div className="app">
       <header className="app-header">
         <img src={logo} className="app-logo" alt="logo" />
         <p>
-          Edit <code>src/App.tsx</code> and save to reload.
+          {t('sample_page.edit')} <code>src/App.tsx</code> {t('sample_page.and_save_to_reload')}.
```

Comment:
Should we change to use [Interpolation](https://www.i18next.com/translation-function/interpolation) in this line? Or I think we can bring `<code>src/App.tsx</code>` to the translations 🤔

Reply:
I don't think we should bring HTML code into the translation, but I see the value of providing an example of "interpolation". So what I can suggest is to remove the "<code>" tag and provide an example of interpolation~

Using that translation: `"message": "Edit {{codeSample}} and save to reload",`

It gives: 

| With code tag | Without Code Tag |
|---|---|
|![image](https://user-images.githubusercontent.com/77609814/143516933-734af4db-6fa1-45de-9ac7-1c0e8a4d7098.png)|![image](https://user-images.githubusercontent.com/77609814/143517071-e099d99e-ab6b-48b9-852c-2fc5baeef095.png)|
|`<p>{t('sample_page.message', { codeSample: '<code>src/App.tsx</code>' })}</p>`|`<p>{t('sample_page.message', { codeSample: '"src/App.tsx"' })}</p>`| 

The `Without code tag` version has been pushed in 1a70ab3 ✅